### PR TITLE
Add knit-footer component

### PR DESCRIPTION
**Pull request by FieryCod**

### CODE REVIEW

File: components/desktop/KnitFooter.vue [hunk 1]
```diff
@@ -1,13 +1,38 @@
 <template>
   <footer class="knit-footer">
-    Footer
+    <a href="/" class="knit-footer__link">
+      <img src="/knit-logotype.svg" class="knit-footer__logo" alt="Logo of Koło Naukowe IT">
```
Comment (zrebcu411) [inline components/desktop/KnitFooter.vue]: When an image is inside a link tag, an `alt` attribute should describe a function of that link, not what this image is. In this example, KNIT logotype links to homepage, not to KNIT logotype. In Polish website, alternative text should be written in polish language also. Please fix it to e.g. 'Koło Naukowe IT'. You can read this article too: https://webaim.org/techniques/alttext/

File: components/desktop/KnitFooter.vue [hunk 2]
```diff
@@ -1,13 +1,38 @@
...
+    </a>
+    <knit-navbar class="navbar--footy"/>
+    <span class="knit-footer__caption"> 2017 &copy; Koło Naukowe IT Politechnika Krakowska </span>
+    <div class="knit-footer__socialbox">
+      <ul class="knit-footer__socialbox-list">
+        <li class="knit-footer__socialbox-item">
+          <a href="/" class="knit-footer__socialbox-link">
+            <i class="flaticon-github-logo" aria-hidden="true"></i>
```
Comment (zrebcu411) [inline components/desktop/KnitFooter.vue]: Icons should not be wrapped in `<i>` tag. Use `span` instead. You can read here about it: https://github.com/w3c/html/issues/732

File: components/desktop/KnitFooter.vue (hunk 1 above)
Comment (FieryCod) [reply components/desktop/KnitFooter.vue]: @zrebcu411 Thanks bro, that link which you've provided is pure gold.

File: components/desktop/KnitFooter.vue (hunk 2 above)
Comment (FieryCod) [reply components/desktop/KnitFooter.vue]: Thanks, fixed :)

**Pull request merged**